developer_docs/NEW_COMMAND.md

```diff
@@ -128,14 +107,13 @@ func (params ListParams) Validate() error {
 ```
 
 Additionally, please create unit tests for your changes. Both the `List()` function and the `Validate()`
-method for `ListParams` need to be tested. These tests should go to `pkg/platform/proxy/proxy_test.go`
-(and `pkg/platform/proxy/proxy_params_test.go`, if you separated the params in their own file).
+method for `ListParams` need to be tested. These tests should go to `cloud-sdk-go/pkg/api/deploymentapi/list_test.go`
 There are many examples of this in the code base, so feel free to browse and use them as inspiration.
 
 That concludes all the steps necessary for creating a new command. You can easily manually test your new command by
-making use of our [helper scripts](../CONTRIBUTING.md#helpers):
+importing your local cloud-sdk-go changes running `make fake-sdk`, and making use of our [helper scripts](../CONTRIBUTING.md#helpers):
 
-`dev-cli --config ~/path/to/your/ecctl/config.yaml platform proxy list`
+`dev-cli --config ~/path/to/your/ecctl/config.yaml deployment list`
```

Comment:
I think this needs to be change to just the file name, say, `config` or `staging` or `prod`, this was wrong to begin with 🙂 .

Reply:
oops! missed that